src/views/Map/Map.tsx

```diff
@@ -36,10 +38,14 @@ const { BivariatePanel } = lazily(() => import('~features/bivariate_manager/comp
 
 const { EventEpisodes } = lazily(() => import('~features/event_episodes'));
 
-type MainViewProps = {
-  userModel: UserDataModel | null;
-};
-export function MainView({ userModel }: MainViewProps) {
+export function MapPage() {
+  const [{ data, loading }] = useAtom(userResourceAtom);
+  const userModel = data && !loading ? data : null;
+
+  useEffect(() => {
```

Comment:
btw, why we init metrics not in CommonView where all common stuff located? We are sure that we won't need metrics on different screens?

Reply:
metrics should init at app start, not when we switch to map page from other page. metrics need to know current route and currently can process only map page